docs/python_docs/python/tutorials/performance/backend/mkldnn/mkldnn_readme.md

```diff
@@ -57,11 +57,24 @@ cd incubator-mxnet
 
 ### Build MXNet with MKL-DNN
 
+To achieve better performance, the Intel OMP and llvm OMP are recommended as below instruction. Otherwise, default GOMP will be used and you may get the sub-optimal performance. If you don't have the full [MKL](https://software.intel.com/en-us/intel-mkl) library installation, you might use OpenBLAS as the blas library, by setting USE_BLAS=openblas.
```

Comment:
is there instruction for how to link llvm omp?

Reply:
OpenMP -> OMP

Reply:
cmake will build and link llvm-omp in 3rdparty/openmp automatically and makefile temporary not support.